changelog/2880.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed Crash while saving Sunpy Maps if they have comments in them
```

Comment:
I would suggest rewording this to, `Fixed io.fits write if the header has the keyword COMMENT`.